oauth_provider/tests/test_rest_api_controller.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import json
+
+from .common_test_controller import OauthProviderControllerTransactionCase
+
+
+class TestRestApiController(OauthProviderControllerTransactionCase):
+
+    API_BASE = '/api/rest/1.0/res.users'
+
+    def setUp(self):
+        super(TestRestApiController, self).setUp('legacy application')
+        self.logged_user = self.env.user
+        self.user = self.env.user
+        self.token = self.new_token().token
+        self.filter.domain = []
+        self.admin_user_data = {"city": "Scranton",
+                                "login": "admin",
+                                "id": self.env.uid,
+                                "name": "Administrator",
+                                "email": "admin@yourcompany.example.com"}
```

Comment:
That's what I'm doing here for the most part, although I think I forgot to reference a few times in the tests. I'll double check